ShopeX/css/main.scss

```diff
@@ -0,0 +1,1037 @@
+@import "colors&fonts";
```

Comment:
Using a & symbol could cause troubles with some systems, for example, Amazon Linux, so better to keep using only alphabet characters. I suggest removing this symbol to reject the change of troubles in the future.